src/components/molecules/FormFields/advancedSettings.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement, useState, FormEvent, ChangeEvent } from 'react'
+import Input from '../../atoms/Input'
+import Button from '../../atoms/Button'
+import { FormContent, FormFieldProps } from '../../../@types/Form'
+import { Field } from 'formik'
+import appConfig from '../../../../app.config'
```

Comment:
always get this within components like so:

```js
const { appConfig } = useSiteMetadata()
```

That's a Gatsby-only thing as we import this into Gatsby's GraphQL layer already on build, so making sure we do not have duplicate json in our final js bundle

Reply:
Thanks for pointing this out, I've updated it